projects/storefrontlib/src/cms-components/organization/budgets/budget-form/budget-form.component.spec.ts

```diff
@@ -202,4 +225,51 @@ describe('BudgetFormComponent', () => {
       expect(component.clickBack.emit).toHaveBeenCalledWith();
     });
   });
+
+  describe('getLocalTimezoneOffset()', () => {
+    it('should return utc-0 offset string', () => {
+      // Use UTC-0 timezone
+      Date.prototype.getTimezoneOffset = () => {
```

Comment:
Modifying the prototype of a global javascript object may potentially affect other tests (*also in different files*).

It would be good to clean up (bring the original prototype in place after the test executes). I can see 2 possible, where I like the second more, as the logic is encapsulated in one place:
1. remember the original Date.prototype.getTimezoneOffset in some variable in `beforeEach` and assign it back to the Date.prototype.getTimezoneOffset in `afterEach`
2. create an util function that does similar thing, but inside itself (no need for before each and after each), i.e.
```typescript
function fakeGetTimezoneOffset(offset: number, callback: Function) {
  const original = Date.prototype.getTimezoneOffset;
  Date.prototype.getTimezoneOffset = () => offset;
  callback();
  Date.prototype.getTimezoneOffset = original;
}
```
```typescript
it('should return utc-0 offset string, fakeLocalTimezoneOffset(0, () => {
   expect(component.getLocalTimezoneOffset()).toEqual('+00:00');
}))
```

What do you think about it?